### ISSUE TITLE
PR 124 doesn't have tf files and template files in sync

### ISSUE THREAD

**Issue by marko7460** [terraform-google-modules/terraform-google-kubernetes-engine#183]
This issue is a consequence of https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/124.

1. Git clone master branch of this repo `git clone git@github.com:terraform-google-modules/terraform-google-kubernetes-engine.git`
2. `cd terraform-google-kubernetes-engine/`
3. `make generate`
4. `git status`

You will see too many files being modified when applying the template:
```
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   README.md
	modified:   cluster_regional.tf
	modified:   cluster_zonal.tf
	modified:   main.tf
	modified:   modules/beta-private-cluster/README.md
	modified:   modules/beta-private-cluster/auth.tf
	modified:   modules/beta-private-cluster/cluster_regional.tf
	modified:   modules/beta-private-cluster/cluster_zonal.tf
	modified:   modules/beta-private-cluster/dns.tf
	modified:   modules/beta-private-cluster/main.tf
	modified:   modules/beta-private-cluster/masq.tf
	modified:   modules/beta-private-cluster/networks.tf
	modified:   modules/beta-private-cluster/outputs.tf
	modified:   modules/beta-private-cluster/sa.tf
	modified:   modules/beta-private-cluster/scripts/delete-default-resource.sh
	modified:   modules/beta-private-cluster/scripts/kubectl_wrapper.sh
	modified:   modules/beta-private-cluster/scripts/wait-for-cluster.sh
	modified:   modules/beta-private-cluster/variables.tf
	modified:   modules/beta-public-cluster/README.md
	modified:   modules/beta-public-cluster/auth.tf
	modified:   modules/beta-public-cluster/cluster_regional.tf
	modified:   modules/beta-public-cluster/cluster_zonal.tf
	modified:   modules/beta-public-cluster/dns.tf
	modified:   modules/beta-public-cluster/main.tf
	modified:   modules/beta-public-cluster/masq.tf
	modified:   modules/beta-public-cluster/networks.tf
	modified:   modules/beta-public-cluster/outputs.tf
	modified:   modules/beta-public-cluster/sa.tf
	modified:   modules/beta-public-cluster/scripts/delete-default-resource.sh
	modified:   modules/beta-public-cluster/scripts/kubectl_wrapper.sh
	modified:   modules/beta-public-cluster/scripts/wait-for-cluster.sh
	modified:   modules/beta-public-cluster/variables.tf
	modified:   modules/private-cluster/README.md
	modified:   modules/private-cluster/cluster_regional.tf
	modified:   modules/private-cluster/cluster_zonal.tf
	modified:   modules/private-cluster/main.tf
	modified:   modules/private-cluster/outputs.tf
	modified:   outputs.tf
	modified:   variables.tf
```

A lot of files have a whitespace or a newline diff, but some files like `modules/beta-private-cluster/cluster_zonal.tf` have diff that shows that `make autogen` was not properly applied in PR124.

**Comment by morgante** [comment]
@marko7460 Could you open a pull request with the drift?

**Comment by marko7460** [comment]
PR: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/184

**Comment by morgante** [comment]
Fixed by #184
